release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -95,6 +95,14 @@ For more information about enabling a feature set during installation, see xref:
 [id="ocp-4-12-post-installation"]
 === Post-installation configuration
 
+==== {product-title} with multi-architecture compute machines (Technology Preview)
+
+{product-title} {product-version} with multi-architecture compute machines now supports manifest listed images on image streams. For more information about manifest list images, see xref:../post_installation_configuration/multi-architecture-configuration.adoc[Configuring multi-architecture compute machines on an {product-title} cluster]. 
```

Comment:
I'm assuming that the "Configuring a multi-architecture cluster" is being revised for this release to include the info about manifest list images as I did not see anything there yet.

Reply:
Oh interesting! I didn't see that before, just needed a rebase, good catch!